x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/list_common/exception_item_card/comments/comments.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import type { EuiCommentProps } from '@elastic/eui';
+import {
+  EuiAccordion,
+  EuiCommentList,
+  EuiFlexItem,
+  EuiPanel,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+
+import * as i18n from '../translations';
+
+export interface ExceptionItemCardCommentsProps {
+  comments: EuiCommentProps[];
+}
+
+export const ExceptionItemCardComments = memo<ExceptionItemCardCommentsProps>(({ comments }) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <EuiFlexItem>
+      <EuiAccordion
+        id="exceptionItemCardComments"
+        buttonContent={
+          <EuiText size="s" style={{ color: euiTheme.colors.primary }}>
+            {i18n.exceptionItemCardCommentsAccordion(comments.length)}
+          </EuiText>
+        }
+        arrowDisplay="none"
+        data-test-subj="exceptionsViewerCommentAccordion"
+      >
+        <EuiPanel
+          data-test-subj="accordion-content"
+          hasBorder={false}
+          hasShadow={false}
+          paddingSize="m"
+        >
+          <EuiCommentList data-test-subj="accordion-comment-list" comments={comments} />
```

Comment:
Super super nit - I've been bad about this and I think originally added lots of these `data-test-subj` but have been trying to update them when I see them to be camel case per Kibana guide - https://github.com/elastic/kibana/blob/8.4/STYLEGUIDE.mdx#camel-case-id-and-data-test-subj

Reply:
Sure, thanks for sharing!